gapic/src/main/com/google/gapid/Main.java

```diff
@@ -218,14 +222,23 @@ private Analytics getAnalytics() {
       return (models == null) ? null : models.analytics;
     }
 
-    private boolean shouldShowUIErrorDialog(Throwable throwable) {
-      // TODO b/178397207: Disable error dialog for a known UI issue, while waiting solution from the SWT side.
-      if (OS.isMac && throwable != null && throwable.getStackTrace().length > 0
-          && "org.eclipse.swt.widgets.Widget".equals(throwable.getStackTrace()[0].getClassName())
-          && "drawRect".equals(throwable.getStackTrace()[0].getMethodName())) {
-        return false;
+    private boolean shouldSuppressUIErrorDialog(Throwable throwable) {
+      if (OS.isMac && throwable != null) {
+
+        // TODO b/178397207: Disable error dialog for a known UI issue, while waiting for asolution from the SWT side.
+        boolean isDrawRectBug = throwable instanceof NullPointerException &&
+                                throwable.getStackTrace().length > 0 &&
+                                "org.eclipse.swt.widgets.Widget".equals(throwable.getStackTrace()[0].getClassName()) &&
+                                "drawRect".equals(throwable.getStackTrace()[0].getMethodName());
+
+        // TODO b/216617761: Disabe error dialog for a different known UI issue, while waiting for a solution from the SWT side.
```

Comment:
Besides the bug number this is the same comment as above, can you update it to be more specific to this one?